Xamarin.Forms.Core/MenuItem.cs

```diff
@@ -49,7 +52,25 @@ public string Text
 			set { SetValue(TextProperty, value); }
 		}
 
-		internal bool IsEnabled
+        /// <summary>
+        /// Setting the background color of the menu item if target platform is iOS.
+        /// </summary>
+        public Color BackgroundColor
```

Comment:
did this ever make to a release? because I don't see a BackgroundColor property on MenuItem in Xamarin.Forms ... :-(